build-images.sh

```diff
@@ -27,6 +27,12 @@ docker tag sauberprojekt/raster_download:$MASTER_TAG sauberprojekt/raster_downlo
 docker build --rm -f "json_download/Dockerfile" -t sauberprojekt/json_download:$MASTER_TAG "json_download"
 docker tag sauberprojekt/json_download:$MASTER_TAG sauberprojekt/json_download:$DATE_TAG
 
+docker build --rm -f "lanuv_download/Dockerfile" -t sauberprojekt/lanuv_download:$MASTER_TAG "lanuv_download"
+docker tag sauberprojekt/lanuv_download:$MASTER_TAG sauberprojekt/lanuv_download:$DATE_TAG
+
```

Comment:
the order of `lanuv` and `lubw` is opposite in above
would better to keep the same order for  consistency